client/src/components/common/Header.js

```diff
@@ -9,12 +9,13 @@ import {
 import PropTypes from 'prop-types'; 
 
 import { loginUser, logoutUser } from "../../actions/actions";
-import RegisterForm from "../forms/RegisterForm";
+import RegisterForm from "../forms/Register";
```

Comment:
"Rename forms/RegisterForm to forms/Register and add refresh" would be a much better commit message.

If 2 commits have the same message, they clearly need to be more descriptive.